src/averageAges.js

```diff
@@ -15,13 +15,24 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const diedInThisCentury = (century)
+    ? people.filter(person => Math.ceil(person.died / 100) === century)
+    : people;
+
+  const onlyMens = getBySex(diedInThisCentury, 'm');
```

Comment:
`men` is plural of `man`
```suggestion
  const onlyMen = getBySex(diedInThisCentury, 'm');
```